src/main/java/Storage.java

```diff
@@ -0,0 +1,64 @@
+package main.java;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * The storage deals with loading tasks from the file and saving tasks in the file
+ *
+ * @author Lio
+ */
+public class Storage {
+    private Scanner data;
+
+    /**
+     * Constructor
+     *
+     * @param filePath File path of the storage file
+     */
```

Comment:
Perhaps the description for the @param should also end with a period, similarly for other JavaDoc comments